cve_bin_tool/cli.py

```diff
@@ -178,7 +201,12 @@ def main(argv=None):
                           """
         LOGGER.warning(warning_nolinux)
 
-    if not os.path.isfile(args.directory) and not os.path.isdir(args.directory):
+    if not args.directory and not args.input_file:
+        LOGGER.error("Error: Either a directory to scan or an input file required")
+        parser.print_usage()
+        return 0
```

Comment:
We should probably be returning an error code instead of 0 here.  NoFileSpecified = -2 maybe?

Reply:
-1 to -5 are already assigned to other exceptions and i believe that it would be nice to keep error code unique. So, maybe we can assign it -6?